src/messages/MOSDBeacon.h

```diff
@@ -39,6 +41,7 @@ class MOSDBeacon : public PaxosServiceMessage {
     if (header.version >= 2) {
       decode(last_purged_snaps_scrub, p);
     }
+    decode(osd_beacon_report_interval, p);
```

Comment:
you need to increase HEAD_VERSION to 3 and only decode this value if header.version >= 3, otherwise set it to utime_t().

Reply:
i think we can set  osd_beacon_report_interval to zero if if header.version < 3.

Reply:
yes, and/or initialize it in the constructor